src/windows/BarcodeScannerProxy.js

```diff
@@ -17,95 +83,105 @@ module.exports = {
      * @param  {array} args       Arguments array
      */
     scan: function (success, fail, args) {
-
         var capturePreview,
             capturePreviewAlignmentMark,
             captureCancelButton,
             capture,
             reader;
-        
+
+        function updatePreviewForRotation() {
```

Comment:
You're using this method as event handler, so probably it makes sense to accept event as parameter here, and use it to detect current orientation instead of querying `DisplayInformation` manually.